components/SmoothLoadingImageFadeTransition.vue

```diff
@@ -0,0 +1,20 @@
+<script>
+</script>
+
+<template>
+  <Transition name="fade">
+    <slot></slot> 
+  </Transition>
+</template>
+
+<style>
+  .fade-enter-active, 
```

Comment:
I'm flipping back and forth if we really need a separate component. I can see the value if we plan to have more effects than just the fade, but we probably won't. 
According to "best practices!": only build for resubility if you know you are going to need it or you already have 3 or more recurring usages. 

also....we could make it conditional css instead of a component as a design alternative 🤔

Reply:
You are the one that had me make a separate `MemeberComponent` for the about us page. That is used only in the About us page. So I was following your previous instructions. The previous standups doctors were wondering if they could use the photo space in the modal. Thought this could be used there as well.